base/iterator.jl

```diff
@@ -348,6 +348,20 @@ length(t::Take) = _min_length(t.xs, 1:t.n, iteratorsize(t.xs), HasLength())
 
 start(it::Take) = (it.n, start(it.xs))
 
+"""
+    take(s::String, n)
+
+Return a `SubString` with at most `n` of the first `Char`s contained in `s`.
```

Comment:
I have a philosophical issue with calling Unicode scalar values "characters". But that seems to be the convention, and I suppose I can raise that issue elsewhere. Will change.